res/schema.xml

```diff
@@ -438,13 +438,4 @@ METADATA
       UPDATE library SET replaygain=0.0 WHERE filetype='flac' COLLATE NOCASE;
     </sql>
   </revision>
-  <revision version="29" min_compatible="3">
```

Comment:
Will removing this cause problems for people who had this change made to their database when the next schema version is merged? Should we leave version 29 as a placeholder that doesn't execute any SQL?

Reply:
Good idea.

Reply:
I'm not clear what you're suggesting. Should I leave this as it is, or make schema version 29 a placeholder?

Reply:
Keeping a placeholder is a good idea. Otherwise beta users will not receive library updates when we alter the schema again later.